src/content/docs/faq.mdx

```diff
@@ -449,11 +451,37 @@ If you'd like to segment availability of Shorebird patches, there are 4 potentia
 
 Yes. Apps built with Shorebird will continue to function normally (as if they had been built without Shorebird), including those that have had patches installed.
 
-### When should I create a patch?
+### When should I create a patch vs a release?
```

Comment:
This will break the link in the concepts section

Reply:
Fixed.